src/Core/Addon/Module/ModuleRepository.php

```diff
@@ -406,7 +407,14 @@ public function getModule($name, $skip_main_class_attributes = false, bool $cach
         return $module;
     }
 
-    public function getModuleAttributes($name)
+    /**
+     * Get Module Attributes (name, displayName, etc.).
+     *
+     * @param string $name The technical name of the module
+     *
+     * @return \Symfony\Component\HttpFoundation\ParameterBag
+     */
```

Comment:
phpdoc comments are already defined in ModuleRepositoryInterface, no need to double the workload.
Best solution is to provide no phpdoc at all in implementation classes, because most IDEs can analyze the interface signature from the implementation.
Or you could refer to interface through documentation inheritance:
```suggestion
    /**
     * {@inheritDoc}
     */
```